### PR TITLE
Task/polling command

### PR DESCRIPTION
**Pull request by dmoranj**
Implements #455

### CODE REVIEW

**Comment by gtorodelvalle** [comment]
LGTM (probably not the best time of the day to do this though... :p)

**Comment by dmoranj** [comment]
You can rethink your support tomorrow, we are not merging this request today.

File: lib/services/groups/groupRegistryMongoDB.js [hunk 1]
```diff
@@ -260,7 +260,7 @@ function init(newConfig, callback) {
 }
 
 function clear(callback) {
-    dbService.dropDatabase(callback);
+    dbService.db.db.dropDatabase(callback);
```
Comment (AlvaroVega) [inline lib/services/groups/groupRegistryMongoDB.js]: is ```db``` duplicated ?

Comment (dmoranj) [inline lib/services/groups/groupRegistryMongoDB.js]: Yes, it is, due to a horrible choice of attribute names (by me). The first db is the attribute of the module containing the connection; the second one is the `db` attribute of the `connection` object, that holds a pointer to the actual database (as can be used in Mongo's command line interpreter).

**Comment by AlvaroVega** [comment]
LGTM

**Pull request merged**